src/components/SearchSploit/SearchSploit.tsx

```diff
@@ -1,137 +1,90 @@
-import { Button, LoadingOverlay, NativeSelect, Stack, TextInput } from "@mantine/core";
+import { Button, LoadingOverlay, NativeSelect, Stack, TextInput, Text, List, Accordion } from "@mantine/core";
 import { useForm } from "@mantine/form";
 import { useCallback, useState } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
-import { Text } from "@mantine/core";
-import { List } from "@mantine/core";
-import { Accordion } from "@mantine/core";
 import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
 
-const title = "SearchSploit";
-const description_userguide =
-    "SearchSploit is a command-line tool used for searching through Exploit-DB that allows an offline " +
-    "copy of the exploited database to be withheld. This tool is useful for conducting security assessments " +
-    "on segregated networks and provides several search options within its operation.\n\nInformation on the " +
-    "tool can be found at: https://www.exploit-db.com/documentation/Offsec-SearchSploit.pdf\n\n" +
-    "Using SearchSploit:\n" +
-    "Step 1: Enter a Search Term followed by selecting a Search Option.\n" +
-    "       Eg: data, Exact\n\n" +
-    "Step 2: Select an Output type.\n" +
-    "       Eg: json\n\n" +
-    "Step 3: Select a Non-Searching option.\n" +
-    "       Eg: Mirror\n\n" +
-    "Step 4: Enter an Exploit Database ID.\n" +
-    "       Eg: PLACEHOLDER (required when using the 'Path' output or Non-Search options)\n\n" +
-    "Step 5: Click Scan to commence SearchSploit's operation.\n\n" +
-    "Step 6: View the Output block below to view the results of the tools execution.";
-
-interface FormValuesType {
+interface FormValues {
     searchTerm: string;
     searchOption: string;
-    out_put: string;
-    non_search: string;
-    ebd_id: string;
+    outputType: string;
+    nonSearch: string;
+    ebdId: string;
 }
 
-const searchOption = ["Case", "Exact", "Strict", "Title"];
-
-const out_put = ["json", "Overflow", "Path", "Verbose", "www"];
-
-const non_search = ["Mirror", "Examine"];
+const TITLE = "SearchSploit";
+const DESCRIPTION = "SearchSploit is a command-line tool used for searching through Exploit-DB.";
+const STEPS = `Step 1: Enter a Search Term followed by selecting a Search Option.
+Step 2: Select an Output type.
+Step 3: Select a Non-Searching option.
+Step 4: Enter an Exploit Database ID.
+Step 5: Click Scan to commence SearchSploit's operation.
+Step 6: View the Output block below to view the results of the tool's execution.`;
+
+const SOURCE_LINK = "https://www.exploit-db.com/documentation/Offsec-SearchSploit.pdf";
+const TUTORIAL = "";
+const SEARCH_OPTIONS = ["Case", "Exact", "Strict", "Title"];
+const OUTPUT_TYPES = ["json", "Overflow", "Path", "Verbose", "www"];
+const NON_SEARCH_OPTIONS = ["Mirror", "Examine"];
 
```

Comment:
You missed dependencies
```suggestion
    const dependencies = ["searchSploit"]; // Contains the dependencies required by the component
```

Reply:
You also miss useEffect

```suggestion
    // Check if the command is available and set the state variables accordingly.
    useEffect(() => {
        // Check if the command is available and set the state variables accordingly.
        checkAllCommandsAvailability(dependencies)
            .then((isAvailable) => {
                setIsCommandAvailable(isAvailable); // Set the command availability state
                setOpened(!isAvailable); // Set the modal state to opened if the command is not available
                setLoadingModal(false); // Set loading to false after the check is done
            })
            .catch((error) => {
                console.error("An error occurred:", error);
                setLoadingModal(false); // Also set loading to false in case of error
            });
    }, []);
```